docs/modelfile.md

```diff
@@ -127,40 +139,54 @@ PARAMETER <parameter> <parametervalue>
 | `{{ .First }}`  | A boolean value used to render specific template information for the first generation of a session.          |
 
 ```
+
 TEMPLATE """
 {{- if .First }}
+
 ### System:
+
 {{ .System }}
 {{- end }}
 
 ### User:
+
 {{ .Prompt }}
 
 ### Response:
+
 """
 
 SYSTEM """<system message>"""
+
 ```
```

Comment:
Whitespace in the TEMPLATE and SYSTEM fields can be significant so I would advise not to modify this

Reply:
my formatter betrayed me, this is fixed now